modules/mono/glue/GodotSharp/GodotSharp/Core/MathfEx.cs

```diff
@@ -1,9 +1,9 @@
 using System;
-
 #if REAL_T_IS_DOUBLE
 using real_t = System.Double;
 #else
 using real_t = System.Single;
+
 #endif
```

Comment:
I don't understand the reason for this empty line addition.

Reply:
It's Rider being dumb. I'm not sure what to do. If I remove it, Rider will just keep adding it next time I format the file 😞

Reply:
You can maybe disable the formatter around this block: https://www.jetbrains.com/help/rider/Enforcing_Code_Formatting_Rules.html#using-comments-to-configure-formatter

Or just leave it be as a quirk yeah. (Or reorder code so that there's a non ifdef'd using coming last.)

Reply:
>(Or reorder code so that there's a non ifdef'd using coming last.)

```cs
#if REAL_T_IS_DOUBLE
using real_t = System.Double;
#else
using real_t = System.Single;
#endif
using System;
```

This seems fine to me if it makes the formatter happy, WDYT @neikeq?

Reply:
I think moving it to the top is the best option. I'm afraid if I add formatter comments for Rider, there could other editors doing the same bad formatting.